editor-components/Image_gallery/image_gallery_7/image-gallery7.tsx

```diff
@@ -0,0 +1,490 @@
+                        key: "card",
+                        displayer: "Card 5",
+                        value: [
+                            { 
+                                type: "image",
+                                key: "image",
+                                displayer: "Image 5",
+                                value: "https://storage.googleapis.com/download/storage/v1/b/hq-composer-0b0f0/o/669e00772f8a5b002ce60145?alt=media", 
+                            },
+                            {
+                                type: "string",
+                                key: "title",
+                                displayer: "title",
+                                value: "John Doe",
+                            },
+                            {
+                                type: "string",
+                                key: "subtitle",
+                                displayer: "subtitle",
+                                value: "Architecture, Interior"
+                            },
+                            {
+                                type:"color",
+                                key:"backgroundColor",
+                                displayer:"Background Color",
+                                value:"#f1f1e8"
+                            }
+                        ]
+                    },
+                    {
+                        type: "array",
+                        key: "card",
+                        displayer: "Card 6",
+                        value: [
+                            { 
+                                type: "image",
+                                key: "image",
+                                displayer: "Image 6",
+                                value: "https://storage.googleapis.com/download/storage/v1/b/hq-composer-0b0f0/o/669e00952f8a5b002ce60151?alt=media", 
+                            },
+                            {
+                                type: "string",
+                                key: "title",
+                                displayer: "title",
+                                value: "Creme",
+                            },
+                            {
+                                type: "string",
+                                key: "subtitle",
+                                displayer: "subtitle",
+                                value: "Architecture, Interior"
+                            }
+                            ,
+                            {
+                                type:"color",
+                                key:"backgroundColor",
+                                displayer:"Background Color",
+                                value:"#f1f1e8"
+                            }
+                        ]
+                    }
+                    ,
+                    {
+                        type: "array",
+                        key: "card",
+                        displayer: "Card 7",
```

Comment:
use singular and non-contain number names. do this for all.
```suggestion
                        displayer: "Card",
```